docs/overview/quick-start.md

```diff
@@ -47,15 +53,11 @@ If you're planning to use [mirrord for Teams](https://app.metalbear.com), you'll
 
 To install the CLI, run:
 
-```bash
-brew install metalbear-co/mirrord/mirrord
-```
-
-or
-
-```bash
-curl -fsSL https://raw.githubusercontent.com/metalbear-co/mirrord/main/scripts/install.sh | bash
-```
+| Operating System | Command |
+| - | - |
+| MacOS | <pre>brew install metalbear-co/mirrord/mirrord</pre> |
```

Comment:
I think brew is supported in Linux as well? Lets have one tab for MacOS/Linux with the first two commands, and one tab for Windows with the Choco command